src/ln/channelmanager.rs

```diff
@@ -1357,17 +1359,17 @@ impl ChannelManager {
 	/// after a PaymentReceived event.
 	/// expected_value is the value you expected the payment to be for (not the amount it actually
```

Comment:
1. The comment for `expected_value` is no longer needed
2. The comment should describe what the function does instead of under what conditions it will be called
3. The comment should describe what the return value represents